src/dotnet/APIView/APIViewWeb/DTOs/SiteNotificationDto.cs

```diff
@@ -1,44 +1,91 @@
-using System;
-using APIViewWeb.Models;
+using System.Text.Json.Serialization;
 
-namespace APIViewWeb.DTOs;
-
-public class SiteNotificationDto
+namespace APIViewWeb.DTOs
 {
-    public string ReviewId { get; set; }
-    public string RevisionId { get; set; }
-    public string Title { get; set; }
-    public string Summary { get; set; }
-    public string Message { get; set; }
-
-    private string _status;
-    public string Status
+    public class SiteNotificationDto
+    {
+        [JsonPropertyName("reviewId")]
+        public string ReviewId { get; set; }
+
+        [JsonPropertyName("revisionId")]
+        public string RevisionId { get; set; }
+
+        [JsonPropertyName("summary")]
+        public string Summary { get; set; }
+
+        [JsonPropertyName("message")]
+        public string Message { get; set; }
+
+        [JsonPropertyName("type")]
+        [JsonConverter(typeof(JsonStringEnumConverter))]
+        public SiteNotificationType Type { get; set; }
+
+        [JsonPropertyName("toastNotification")]
+        public ToastNotificationDto ToastNotification { get; set; }
```

Comment:
Do you really need to add ToastNotification here? I think it better to decide that on the client side.